Formula/flow.rb

```diff
@@ -1,8 +1,10 @@
 class Flow < Formula
   desc "Static type checker for JavaScript"
   homepage "https://flowtype.org/"
-  url "https://github.com/facebook/flow/archive/v0.26.0.tar.gz"
-  sha256 "7813c98f7509d89e5a187df4252dbf8e6c429b1d711e10156eccb1ac793fb571"
+  url "https://github.com/facebook/flow.git",
+    :tag => "v0.27.0",
+    :revision => "ceea4c3616824b9ad7cd7028dece2d55a7f3b539"
+  sha256 "c2e2b9d398815825a7bf06c019652a3e3daa8d859e939dbf1d8ed6cb03635462"
```

Comment:
You can scrap the `sha256` if we're using a git checkout. The `:revision` serves as the checksum in this case. Possibly worth adding a short note like:

```ruby
# Switch back to archive tarball when possible.
# link/to/upstream/bug/report
```